src/get-started/java-binding.md

```diff
@@ -1312,7 +1312,7 @@ For using the library just include the dependency in your `pom.xml`:
 [homebrew]: https://github.com/Homebrew/brew#homebrew
 [how-to-build]: https://github.com/exonum/exonum-java-binding/blob/ejb/v0.9.0-rc2/CONTRIBUTING.md#how-to-build
 [java.util.Properties]: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Properties.html#load(java.io.Reader)
-[jdk]: https://jdk.java.net/
+[jdk]: https://jdk.java.net/java-se-ri/11
```

Comment:
The reference implementation is not something we shall link:

> These binaries are for reference use only!

> These binaries are provided for use by implementers of the Java SE 11 Platform Specification and are for reference purposes only

I think we shall either drop the link; or link one of the providers, e.g., https://adoptopenjdk.net/?variant=openjdk11&jvmVariant=hotspot (but that is certainly not the only provider)

Reply:
Yep, it was my concern